coriolisclient/v1/migrations.py

```diff
@@ -48,7 +48,8 @@ def get(self, migration):
 
     def create(self, origin_endpoint_id, destination_endpoint_id,
                destination_environment, instances, network_map=None,
-               storage_mappings=None, skip_os_morphing=False):
+               storage_mappings=None, source_environment=None,
```

Comment:
IMO `source_environment` should be a required positional parameter (right before the `destination_environment`) in the client method headers themselves, and should `None` be passed in, it should *NOT* be included in the API body at all.